packages/project-disaster-trail/src/state/tasks.js

```diff
@@ -14,15 +14,24 @@ const initialState = {
   activeEnvironment: defaultEnv,
   taskOrder: shuffle(defaultSaveYourself),
   activeTask: 0,
-  completedTasks: []
+  // activeTime: 0,
+  // startTime: 0,
+  // totalTime: 0,
+  completedTasks: [],
+  outOfTime: false
+  // playing: true
 };
 
+// const timerId = 0;
```

Comment:
It looks like we can remove all of this commented out stuff now

Reply:
You got it!